apps/frontend/src/pages/onboarding/styles.module.css

```diff
@@ -42,16 +42,28 @@
 	margin-bottom: auto;
 }
 
+.answers label {
+	text-align: center;
+}
+
 .button-container {
 	display: flex;
+	flex-direction: column;
 	gap: 15px;
 	align-items: center;
-	justify-content: center;
 	width: 100%;
-	max-width: 300px;
 	margin-top: 76px;
 }
 
+.button-container button {
```

Comment:
B10. No global tag styles. Only class selectors or a cascade are used to set styles.

Reply:
I don't think its a global tag style because I'm only targeting button inside the button-container.

Reply:
I think Julia is right. We should use only class selectors. Look at B10, there is example for this case.